src/endpoints/getResults.ts

```diff
@@ -10,19 +10,25 @@ import { ContentFilter } from '../enums/ContentFilter'
 type GetResultsArguments =
   | {
       pages?: number
+      startPage?: number
+      endPage?: number
       teamID?: number
       eventID?: never
       contentFilters?: ContentFilter[]
     }
   | {
       pages?: never
+      startPage?: number
```

Comment:
I would remove the `pages` option since `pages: 20` is the same as `startPage: 0, endPage: 20`

Reply:
done